certification/internal/policy/less_than_max_layers.go

```diff
@@ -1,27 +1,28 @@
 package policy
 
-import "github.com/komish/preflight/certification/errors"
+type UnderLayerMaxPolicy struct {
+}
 
-func UnderLayerMax() *Definition {
-	return &Definition{
-		ValidatorFunc: isUnderLayerMaxValidatorFunc,
-		Metadata:      isUnderLayerMaxPolicyMeta,
-		HelpText:      isUnderLayerMaxPolicyHelp,
-	}
+func (p UnderLayerMaxPolicy) Validate(image string) (bool, error) {
+	return true, nil
 }
```

Comment:
While these are all unimplemented, can these all return `false` and the Err indicating they're not implemented?